ogcapi-stable/ogcapi-foundation/src/main/java/de/ii/ogcapi/foundation/infra/rest/ExceptionMapper.java

```diff
@@ -110,7 +110,7 @@ public Response toResponse(Throwable exception) {
       return processException(
           exceptionFormat, msgCause.isEmpty() ? msg : String.format("%s: %s", msg, msgCause));
     } else if (exception instanceof VolatileUnavailableException) {
-      return serviceUnavailable(exceptionFormat);
+      return serviceUnavailable(exceptionFormat, exception.getMessage());
```

Comment:
This will contain internal error messages, they might be logged but should not be shown to the user.
These exceptions are thrown when something unavailable is accessed before the state change was detected, e.g. when the database becomes unavailable between polling intervals.
```suggestion
      return serviceUnavailable(exceptionFormat, null);
```